modules/platforms/dotnet/Apache.Ignite.Core/Client/IIgniteClient.cs

```diff
@@ -111,6 +112,11 @@ public interface IIgniteClient : IDisposable
         [SuppressMessage("Microsoft.Design", "CA1024:UsePropertiesWhereAppropriate", Justification = "Semantics.")]
         IBinary GetBinary();
 
+        /// <summary>
+        /// Gets Ignite transactions facade.
+        /// </summary>
+        IClientTransactions Transactions { get; }
```

Comment:
Please convert to a method for consistency with `IIgnite.GetTransactions`.
Method is also a more future-proof option (see https://docs.microsoft.com/en-us/previous-versions/dotnet/netframework-4.0/ms229054(v=vs.100)).